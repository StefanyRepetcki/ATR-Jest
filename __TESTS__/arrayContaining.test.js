// https://jestjs.io/pt-BR/docs/expect#expectarraycontainingarray

const users = require("../modulo-3-4/aula2/users");
const usersAnotherArray = ["JosÃ©", "Maria"];

describe("Checking items of array", () => {
    test("should check if the array contains all the items in another array", () => {
        expect(users).toEqual(expect.arrayContaining(usersAnotherArray));
    });
});