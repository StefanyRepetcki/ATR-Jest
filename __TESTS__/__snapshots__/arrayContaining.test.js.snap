// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking items of array should check if the array contains all the items in another array 1`] = `
[
  "Thomas",
  "Maria",
  "Jos√©",
]
`;
